Editor/Resources/EditorWindow/Components/ProfileSelector.uxml

```diff
@@ -0,0 +1,22 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <ui:Style src="../common.uss"/>
+    <ui:VisualElement class="form-row">
+        <ui:Label name="DropdownLabel" class="form-row__label"/>
+        <ui:DropdownField name="Dropdown" class="dropdown form-row__input"/>
+    </ui:VisualElement>    
+    <ui:VisualElement class="form-row">
+        <ui:Label name="BucketNameLabel" class="form-row__label"/>
+        <ui:Label name="BucketName" class="form-row__text-display"/>
+    </ui:VisualElement>
+    <ui:VisualElement class="form-row">
+        <ui:Label name="RegionLabel" class="form-row__label"/>
+        <ui:Label name="Region" class="form-row__text-display"/>
+    </ui:VisualElement>
+    <ui:VisualElement class="form-row">
+        <ui:Label name="BootstrapStatusLabel" class="form-row__label"/>
+        <ui:Label name="BootstrapStatus" class="form-row__text-display"/>
```

Comment:
We don't appear to be populating these values. Please add that with this PR so that this component/PR is self-contained

Reply:
Populated from statemanager